pages/agent/v3/tokens.md

```diff
@@ -1,100 +1,126 @@
+
+        ```curl
+        curl -H "Authorization: Bearer $TOKEN" "https://api.buildkite.com/v2/organizations/{org.slug}/clusters"
+        ```
+
 <!--alex ignore clearly-->
 
-The token description should clearly identify the environment the token is intended to be used for, and is shown on your [Agents page](https://buildkite.com/organizations/-/agents) (for example, `Read-only token for static site generator`).  
+- The `description` value clearly identifies the environment the token is intended to be used for (for example, `Read-only token for static site generator`), and is listed on the _Agent tokens_ page of your specific cluster the agent connects to. To access this page, select _Agents_ > the specific cluster > _Agent Tokens_.
 
-It is possible to create multiple agent tokens using the GraphQL API. These tokens will show up on the [Agents page](https://buildkite.com/organizations/-/agents) in the UI, but can only be managed (created or revoked) using the API.
+## Revoke a token
 
-## Revoking tokens
+Agent tokens can be revoked using the [_Agent Tokens_ page of a cluster](#revoke-a-token-using-the-buildkite-interface), or via the [REST API's delete agent token](#revoke-a-token-using-the-rest-api) feature.
 
-Tokens can be revoked using the [GraphQL API](/docs/apis/graphql-api) with the `agentTokenRevoke ` mutation.
+Once a token is revoked, no new agents will be able to start with that token. Revoking a token does not affect any connected agents.
 
-You need to pass your agent token as the ID in the mutation. You can get the token from your Buildkite dashboard, in _Agents_ > _Reveal Agent Token_, or you can retrieve a list of agent token IDs using this query:
+### Using the Buildkite interface
 
-```graphql
-query GetAgentTokenID {
-  organization(slug: "organization-slug") {
-    agentTokens(first:50) {
-      edges {
-        node {
-          id
-          uuid
-          description
-        }
-      }
-    }
-  }
-}
-```
+To revoke a cluster's agent token using the Buildkite interface:
+
+1. Select _Agents_ to access the _Agent Clusters_ page.
+1. Select the cluster containing the agent token to revoke.
+1. Select _Agent Tokens_ and on this page, expand the agent token to revoke.
+1. Select _Revoke_ > _Revoke Token_ in the confirmation message.
+
+### Using the REST API
 
-Then, using the token ID, revoke the agent token:
-
-```graphql
-mutation {
-  agentTokenRevoke(input: {
-    id: "token-id",
-    reason: "A reason"
-  }) {
-    agentToken {
-      description
-      revokedAt
-      revokedReason
-    }
-  }
-}
+To [revoke (that is, delete) an agent token](/docs/apis/rest-api/clusters#agent-tokens-delete-a-token) using the [REST API](/docs/apis/rest-api), run the following example `curl` command:
```

Comment:
This is the only place where `(that is, delete`) is added as explanation. Suggest removing here, or considering where else to have this hint (should it be higher up on the page as well?)

Reply:
Yeah, I was feeling the same way about this too. I decided to change this to just `(delete)` because the linked docs for the API call is titled 'delete a token' (and the API call uses the 'delete' method). However, on 2nd thoughts, I'm thinking I should change the title of this API call section of the docs to 'revoke a token' too. Let me double-check on Monday.

Reply:
I'm changing 'delete a token' to 'revoke a token'.